azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/CallCompositeException.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Defines the base type of custom Exception that can be thrown by this Library.
+ */
+public class CallCompositeException extends RuntimeException {
```

Comment:
@Albert Lo do we have discussion with iOS about introducing new exception? I believe this change will also require Internal and Azure API review?

Reply:
@iaulakh I will send a message to the group chat to solicit feedback